src/config/inversion-fixes.config

```diff
@@ -1775,6 +1775,7 @@ INVERT
 NO INVERT
 .mwe-math-fallback-image-inline
 .mwe-math-fallback-image-display
+.mwe-popups * image
```

Comment:
This selector looks strange. Your selector is read as `SVG image inside any element that is inside element with class="mwe-popups"`. Maybe you can remove `*`, it will  make it more performant.

Reply:
For some reason I thought that without the `star` the selector would match only `image` elements that are _immediate_ descendants of elements with the class `mwe-popups`.

Reply:
`.mwe-popup > image` means immediate descendant.